sdk/python/tests/compiler/testdata/conditions_and_loops.yaml

```diff
@@ -15,103 +15,49 @@
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  name: conditions-and-loops
   annotations:
-    tekton.dev/output_artifacts: '{"add-numbers": [{"key": "artifacts/$PIPELINERUN/add-numbers/Output.tgz",
-      "name": "add-numbers-Output", "path": "/tmp/outputs/Output/data"}], "print-number":
-      [{"key": "artifacts/$PIPELINERUN/print-number/Output.tgz", "name": "print-number-Output",
-      "path": "/tmp/outputs/Output/data"}], "produce-numbers": [{"key": "artifacts/$PIPELINERUN/produce-numbers/Output.tgz",
-      "name": "produce-numbers-Output", "path": "/tmp/outputs/Output/data"}]}'
-    tekton.dev/input_artifacts: '{"print-number": [{"name": "add-numbers-Output",
-      "parent_task": "add-numbers"}]}'
+    pipelines.kubeflow.org/pipeline_spec: '{"inputs": [{"default": "3", "name": "n",
+      "optional": true}, {"default": "20", "name": "threshold", "optional": true}],
+      "name": "Conditions and loops"}'
+    sidecar.istio.io/inject: 'false'
     tekton.dev/artifact_bucket: mlpipeline
     tekton.dev/artifact_endpoint: minio-service.kubeflow:9000
     tekton.dev/artifact_endpoint_scheme: http://
     tekton.dev/artifact_items: '{"add-numbers": [["Output", "$(results.output.path)"]],
       "notify-failure": [], "notify-success": [], "print-number": [["Output", "$(results.output.path)"]],
       "produce-numbers": [["Output", "$(results.output.path)"]]}'
-    sidecar.istio.io/inject: "false"
-    pipelines.kubeflow.org/pipeline_spec: '{"inputs": [{"default": "3", "name": "n",
-      "optional": true}, {"default": "20", "name": "threshold", "optional": true}],
-      "name": "Conditions and loops"}'
+    tekton.dev/input_artifacts: '{"print-number": [{"name": "add-numbers-Output",
+      "parent_task": "add-numbers"}]}'
+    tekton.dev/output_artifacts: '{"add-numbers": [{"key": "artifacts/$PIPELINERUN/add-numbers/Output.tgz",
+      "name": "add-numbers-Output", "path": "/tmp/outputs/Output/data"}], "print-number":
+      [{"key": "artifacts/$PIPELINERUN/print-number/Output.tgz", "name": "print-number-Output",
+      "path": "/tmp/outputs/Output/data"}], "produce-numbers": [{"key": "artifacts/$PIPELINERUN/produce-numbers/Output.tgz",
+      "name": "produce-numbers-Output", "path": "/tmp/outputs/Output/data"}]}'
+  name: conditions-and-loops
 spec:
   params:
-  - name: "n"
+  - name: n
```

Comment:
The yaml validator in the UI has issues with `n` being used as the name. Can we change `n` to `num` or something else in  `sdk/python/tests/compiler/testdata/conditions_and_loops.py`